src/evaluators/state/PHAL_SaveSideSetStateField_Def.hpp

```diff
@@ -59,6 +59,8 @@ SaveSideSetStateField (const Teuchos::ParameterList& p,
 
   savestate_operation = Teuchos::rcp(new PHX::Tag<ScalarT>(fieldName, dl->dummy));
 
+  useCollapsedSidesets = dl->useCollapsedSidesets;
+
   this->addDependentField (field.fieldTag());
   this->addEvaluatedField (*savestate_operation);
 
```

Comment:
@mcarlson801  The two TEUCHOS_TEST_FOR_EXCEPTION below, need to be modified for collapsed side sets (`.size() < 2` and `.name(1) `)